src/site/_ru/_betterbook.yaml

```diff
@@ -57,9 +57,9 @@ toc:
         path: /fundamentals/discovery-and-distribution/
         id: discovery-and-distribution
         section:
-          - title: "Optimization for Crawlers"
+          - title: "Оптимизация под поисковых ботов"
             path: /fundamentals/discovery-and-distribution/optimization-for-crawlers/
-          - title: "Enriching Search Results"
+          - title: "Улучшение результатов поиска"
```

Comment:
Looking at the translation it says "improving", I think this is not correct, we are not improving the placing of search results, we are making your site look better.